_posts/2025-04-06-sm9276.md

```diff
@@ -0,0 +1,84 @@
+---
+
+layout: post
+
+# If your post title is longer or more complicated
+
+# than can be represented in the filename, uncomment the following line
+
+# and specify a custom title
+
+# title: "Sample Blog Post"
+
+categories: 
+
+- Contribution
+
+# Enter your name below.
+
+author: Sergio Mercado Nunez
+
+---
+
+## Introduction to the Community
+
+For my first open-source contribution, I chose to work with the [Profanity](https://github.com/profanity-im/profanity) project, which is a lightweight,
+
+terminal-based XMPP client written in C. 
+
+I chose Profanity because I found the project approachable. I initially struggled with picking a project. 
+
+as many of them seemed very overwhelmed with the code bases. 
+
+I decided to do documentation because I had a lot of trouble looking for a project to contribute to; I was too intimidated by the code most of the time. I realized that most of the
+
+projects I could just try to understand and document the code, which will make it easier for myself and others to try and contribute to projects.
+
+## Why I Picked It
+
+The reason I picked Profanity is because I am super interested in XMPP as a protocol; it seems so awesome and secure. I wanted to work.
+
+on a project that I would use, so I set up an XMPP account so I can use it with this client. I saw that the program was written in C, which
+
+is a language I am comfortable in. 
+
```

Comment:
is there a reason for the gaps in the sentence like this? this is causing the sentence to be broken up on multiple lines